src/routes/public/stocks.routes.js

```diff
@@ -0,0 +1,14 @@
+import { Router } from "express";
+import {
+  getARandomStock,
+  getStockById,
+  getStocks,
+} from "../../controllers/public/stocks.controllers.js";
+
+const router = Router();
+
+router.route("/").get(getStocks);
+router.route("/:stockId").get(getStockById);
```

Comment:
Based on the `nse-stock.json` file we are filtering `getStockById` based on `Symbol` key in the json object. So, It would be great if we use`:stockSymbol` as a parameter placeholder instead of `:stockId` which is more readable and easy to understand on which field to use as a parameter.

Reply:
done